Barotrauma/BarotraumaShared/SharedSource/Characters/Character.cs

```diff
@@ -87,6 +87,26 @@ public bool IsRemotelyControlled
             }
         }
 
+        public Client GetCharacterClient
+        {
+            get
+            {
+                dynamic controller = null;
```

Comment:
Why the controller needs to be dynamic? Dynamic is used when the type may change or when it's unknown. Here it's Client. I'm sure there's a place for that in some applications, scripting etc, but personally I've never found any reason to use that keyword, because static typing and compile time errors are generally preferable over dynamic types and runtime errors.

Reply:
Changed it to "Client" instead of using dynamic.